tasks/task2/README.md

```diff
@@ -1,25 +1,63 @@
-# Store
-Let's imagine that we are building the store app.
-It should be the simple store where we can just manage orders and products (as admins).
-Customers should be able to place the order.
+# Store App Task
+
+## Overview
+
+Welcome to the Store App task!
+In this exercise, you will be developing a simple store application where admins can manage products and orders, while customers can place orders.
```

Comment:
🤔 i believe we do not have any tasks related to managing orders by admins, so maybe we can remove the `and orders` from this sentence?